application/models/Permission.php

```diff
@@ -720,15 +721,15 @@ private static function comparePermissionTitle($aApermission, $aBpermission)
     public function getUserId($iUserID = null)
```

Comment:
As you are editing this, I wonder, do you know what this is used for? Somehow I think we need to add a `@todo`  here for replacing its usage, in the future, for App->getCurrentUserId().

I think I have missed this comment on the master branch. Maybe the `@todo` is even more important to be added there. What do you think? Just a comment, not request.

Reply:
@deprecated ? And update usage directly ?

In master only then.